external/radiation/tests/test_wrapper_api.py

```diff
@@ -83,3 +89,104 @@ def test__is_compute_timestep(time_index, compute_period, expected_compute):
 def test__solar_hour(time, init_time, expected_solar_hour):
     solar_hour = _solar_hour(time, init_time)
     assert np.isclose(solar_hour, expected_solar_hour)
+
+
+class DummyComm:
+    rank: int = 0
+
+    def __init__(self):
+        pass
+
+
+RADIATION_KWARGS = {
+    "tracer_inds": {
+        "specific_humidity": 1,
+        "cloud_water_mixing_ratio": 2,
+        "rain_mixing_ratio": 3,
+        "cloud_ice_mixing_ratio": 4,
+        "snow_mixing_ratio": 5,
+        "graupel_mixing_ratio": 6,
+        "ozone_mixing_ratio": 7,
+        "cloud_amount": 8,
+    },
+    "comm": DummyComm(),
+    "timestep": 900.0,
+    "init_time": datetime.datetime(2016, 8, 1, 0, 0, 0),
+}
+
+
+def test_config_validation(regtest):
+    rad_config = RadiationConfig()
+    radiation_wrapper = Radiation(rad_config, **RADIATION_KWARGS)
+    with regtest:
+        radiation_wrapper.validate()
+
+
+@pytest.mark.parametrize(
+    ["rad_config_kwargs"],
+    [
+        pytest.param({"iovrsw": 2}, id="max_overlap_not_implemented"),
+        pytest.param({"isolar": 5}, id="invalid_solar_constant"),
+    ],
+)
+def test_wrapper_raises_on_invalid(rad_config_kwargs):
+    """Doesn't make sense to test all of the config validation, but
+        try a couple known invalid combinations to make sure they fail.
+    """
+    rad_config = RadiationConfig(**rad_config_kwargs)
+    with pytest.raises(ValueError):
+        Radiation(rad_config, **RADIATION_KWARGS).validate()
+
+
+def test_gfs_physics_control_config_raises_on_invalid():
```

Comment:
Will delete this since it duplicates `test_levr_different_error` in `test_config`.